scripts/build_kernel.sh

```diff
@@ -45,17 +57,18 @@ export DEBIAN_KERNEL_DISABLE_DEBUG=
 _source_dir=$(find .  -maxdepth 1 -type d -name "linux-*")
 cd $_source_dir
 
-if find ./debian/config/arm64/rpi/ -name "config.v8" -printf 1 -quit | grep -q 1
+if find ./debian/config/arm64/rpi/ -name "config.$FLAVOUR" -printf 1 -quit | grep -q 1
 then
-    cp ../scripts/v8-config-overlay ./debian/config/arm64/rpi/config.v8
+    cp ../scripts/"$FLAVOUR"-config-overlay ./debian/config/arm64/rpi/config."$FLAVOUR"
 else
-    echo V8 Configuration file not found. Perhaps the source structure has changed?
+    echo "$FLAVOUR" Configuration file not found. Perhaps the source structure has changed?
     exit 1
 fi
 
-make -f ./debian/rules.gen binary-arch_${ARCH}_${FEATURESET}_${FLAVOUR}
-mv /opt/kernel/*.deb /output
+make -f ./debian/rules.gen binary-arch_"$ARCH"_"$FEATURESET"_"$FLAVOUR"
+mkdir -p /output/"$FLAVOUR"
+mv /opt/kernel/*.deb /output/"$FLAVOUR"
 
 # Update the package index
-cd /output
+cd /output/"$FLAVOUR"
```

Comment:
This folder is meant to be served via the apache2 container and act as a new apt repositories to be added to the RaspiOS apt config (in the `/etc/apt/sources.list.d` directory)

Now if we run `dpkg-scanpackages` in each flavour directory we'll need to configure an apt source entry for each variant.

As all the variants are basically the same package, keeping them in a single apt repository would be easier to maintain.

Can you put all the packages in the same directory without flavour?

Reply:
This has been addressed ✅